test/config_parser_SUITE.erl

```diff
@@ -508,11 +509,12 @@ listen_c2s_fast_tls(_Config) ->
     T = fun(Opts) -> listen_raw(c2s, #{<<"port">> => 5222,
                                        <<"tls">> => Opts}) end,
     P = [listen, 1, tls],
-    ?cfg(P, default_c2s_tls(fast_tls), T(#{})),
```

Comment:
Maybe we could add a test equivalent to what we do in `test_just_tls_common/2`, which would check that `missing_cacertfile` is thrown if `cacertfile` is missing, i.e. in case of `T(#{})` or `T(#{verify_mode => peer})` but not for `T(#{verify_mode => none})`? Unless it is checked somewhere else, and I missed that.

Reply:
Added